README.md

```diff
@@ -44,7 +44,7 @@
 
 <h2 align="center">From zero to hero</h2>
 
-Texthero is a python toolkit to work with text-based dataset quickly and effortlessly. Texthero is very simple to learn and designed to be used on top of Pandas. Texthero has the same expressiveness and power of Pandas and is extensively documented. Texthero is modern and conceived for programmers of the 2020 decade with little knowledge if any in linguistic. 
+Texthero is a python toolkit to work with text-based dataset quickly and effortlessly. Texthero is very simple to learn and designed to be used on top of Pandas. Texthero has the same expressiveness and power of Pandas and is extensively documented. Texthero is modern and conceived for programmers of the 2020 decade with little knowledge if any in linguistic.
 
 You can think of Texthero as a tool to help you _understand_ and work with text-based dataset. Given a tabular dataset, it's easy to _grasp the main concept_. Instead, given a text dataset, it's harder to have quick insights into the underline data. With Texthero, preprocessing text data, mapping it into vectors, and visualizing the obtained vector space takes just a couple of lines.
 
```

Comment:
Maybe

Texthero includes toots for:
- Preprocessing text data: clean and tokenize texts with support for both out-of-the-box and custom solutions.
- Text Representation: vectorization with TF-IDF / Term Frequency / Count, and custom word embeddings (wip)
- Vector space analysis: dimensionality reduction (PCA, NMF, t-SNE) and clustering ...

Reply:
done :octocat: